operations/src/main/java/no/sikt/nva/handle/CreateHandleHandler.java

```diff
@@ -0,0 +1,88 @@
+package no.sikt.nva.handle;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.function.Supplier;
+import no.sikt.nva.handle.exceptions.CreateHandleException;
+import no.sikt.nva.handle.exceptions.MalformedRequestException;
+import no.sikt.nva.handle.model.HandleRequest;
+import no.sikt.nva.handle.model.HandleResponse;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+
+import java.net.HttpURLConnection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.util.Objects.isNull;
+import static no.sikt.nva.handle.HandleDatabase.ERROR_CREATING_HANDLE_FOR_URI;
+import static no.sikt.nva.handle.utils.DatabaseConnectionSupplier.getConnectionSupplier;
+
+public class CreateHandleHandler extends ApiGatewayHandler<HandleRequest, HandleResponse> {
```

Comment:
No tests?

Reply:
operations/src/test/java/no/sikt/nva/handle/CreateHandleHandlerTest.java

or where you thinking about something else?

Reply:
Thought it was a new class but could not find a new test class. I assume its already in the sam template as well?

Reply:
yup :-) just me that messed up refactoring. tried a desperate rebase here to spilt the move commit, but did not manage in a reasonable timeframe